app/components/layout/top-nav.tsx

```diff
@@ -117,3 +125,26 @@ function LogoutButton({ className }: { className?: string }) {
     </Form>
   );
 }
+
+export function TopNavUserMenu() {
+  return (
+    <div>
+      <StyledMenu
+        button={
+          <div className="w-8">
+            <UserCircleIcon />
+          </div>
+        }
+      >
+        <>
+          <Menu.Item>
+            <Link to="/settings">Settings</Link>
+          </Menu.Item>
+          <Menu.Item>
+            <LogoutButton />
+          </Menu.Item>
+        </>
+      </StyledMenu>
+    </div>
```

Comment:
for some reason, the div is needed, if not it behaves differently where the button gets displaced

Reply:
Weird. Maybe we should leave an inline comment so other devs in the future will know to not remove the `div` on accident?